test/Rize/Uri/Node/ParserTest.php

```diff
@@ -85,4 +86,45 @@ public function testParseTemplate()
 
         $this->assertEquals($expected, $actual);
     }
-}
\ No newline at end of file
+
+    public function testParseTemplateWithLiteral()
+    {
+        // will pass
+        $uri = new UriTemplate('http://www.example.com/v1/company/', []);
```

Comment:
ah good spot will fix that, also there is another issue with composer running on PHP 5.3.3 on Travis but I believe that is a generic problem with their container:.